service/resource/clusterrolebindingv2/resource.go

```diff
@@ -0,0 +1,107 @@
+package clusterrolebindingv2
+
+import (
+	"reflect"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	apiv1 "k8s.io/api/rbac/v1beta1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "clusterrolebindingv2"
+
+	VersionBundleVersionAnnotation = "giantswarm.io/version-bundle-version"
```

Comment:
When this is used in several places we should put it into the key package. This was originally supposed to be used in the deployment resource only.